modules/cli-options/src/main/scala/scala/cli/commands/PackageOptions.scala

```diff
@@ -39,6 +39,12 @@ final case class PackageOptions(
   @HelpMessage("Generate an assembly JAR")
     assembly: Boolean = false,
   @Group("Package")
+  @HelpMessage("For assembly JAR, whether to add a bash / bat preamble")
```

Comment:
Shouldn't we add using directives for those options? Especially for `provided` one

Reply:
Should be fixed.